packages/token/index.ts

```diff
@@ -1,18 +1,17 @@
 import type { HexString } from "~typing"
 
-export class Token {
-  public constructor(
-    public readonly address: HexString,
-    public readonly symbol: string,
-    public readonly decimals: number
-  ) {}
+export type Token = {
+  address: HexString
+  symbol: string
+  decimals: number
 }
 
-// TODO: Network specific
-export const ETH = new Token(
-  "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
-  "ETH",
-  18
-)
+export type AccountToken = Token & {
+  balance: HexString
+}
```

Comment:
I think it is not suitable to put at here. For token package it should not be aware of account relevant stuffs. A compromising way is to put this type to storage, and leverage type in token package.